bluebinder.c

```diff
@@ -10,6 +10,7 @@
  *  Based on bluez5/tools/btproxy.c:
  *  Copyright (C) 2011-2012  Intel Corporation
  *  Copyright (C) 2004-2010  Marcel Holtmann <marcel@holtmann.org>
+ *  Copyright (C) 2022 Open Mobile Platform LLC.
```

Comment:
I believe if this is to be added it should be under line 8 (`Copyright (C) 2018-2022  Jolla Ltd.`), not under the copyright of `bluez5/tools/btproxy.c` this code is based on.

Reply:
The indent is also wrong.

Reply:
thanks, fixed it